drizzlepac/haputils/poller_utils.py

```diff
@@ -426,13 +445,23 @@ def parse_mvm_tree(det_tree, log_level):
                 #
                 # svm prod_info = 'skycell_p1234_x01y01 wfc3 uvis f200lp all 2009 1 drz'
```

Comment:
Is this "svm prod_info" or "mvm prod_info"?

Reply:
Done.  Resolved in 21fa202.